src/main/kotlin/app/extractors/SwiftExtractor.kt

```diff
@@ -4,35 +4,7 @@
 
 package app.extractors
 
-class SwiftExtractor : ExtractorInterface {
-    companion object {
-        const val LANGUAGE_NAME = Lang.SWIFT
-        val importRegex = Regex("""^(.*import)\s[^\n]*""")
-        val commentRegex = Regex("""^([^\n]*//)[^\n]*""")
-        val extractImportRegex = Regex("""import\s+(\w+)""")
-    }
-
-    override fun extractImports(fileContent: List<String>): List<String> {
-        val imports = mutableSetOf<String>()
-
-        fileContent.forEach {
-            val res = extractImportRegex.find(it)
-            if (res != null) {
-                val lineLib = res.groupValues[1]
-                imports.add(lineLib)
-            }
-        }
-
-        return imports.toList()
-    }
-
-    override fun tokenize(line: String): List<String> {
-        var newLine = importRegex.replace(line, "")
-        newLine = commentRegex.replace(newLine, "")
-        return super.tokenize(newLine)
-    }
-
-    override fun getLanguageName(): String? {
-        return LANGUAGE_NAME
-    }
-}
+object SwiftExtractor : ExtractorBase(
+    language = Lang.SWIFT,
+    importRegex = Regex("""^(.*import)\s[^\n]*"""),
```

Comment:
`extractImportRegex` used for actually extracting of imports for choosing right classifiers
`importRegex` used to filter tokens from imports and as @yaronskaya works slightly faster

Here importRegex (less accurate version) used for both extracting and filter tokens. @yaronskaya review this change.

Reply:
@asurkov this regex doesn't extract imports. Please, use 'extractImportRegex' from previous version of SwiftExtractor. I wonder if it passes tests:)